package.json

```diff
@@ -84,7 +84,7 @@
     "coverage": "gulp coveralls",
     "generate-docs": "node ./build/commands-yml/parse.js",
     "clean": "rm -rf node_modules && rm -rf package-lock.json && npm install",
-    "clean:prod": "rm -rf node_modules && rm -rf package-lock.json && npm install --production",
+    "shrinkwrap-prod": "rm -rf package-lock.json && npm prune --production && npm shrinkwrap",
```

Comment:
did you mean to delete clean:prod?

Reply:
@jlipps Yeah, it wasn't being used anywhere.